src/api/registration/registration.double.js

```diff
@@ -0,0 +1,20 @@
+const doubleResponse = {
+  "fsa-rn": "DOUBLE-TEST"
+};
+
+const registrationDouble = (doubleMode) => {
```

Comment:
do we actually use double mode much? Really tempted to strip them all out at some point if not

Reply:
Can be useful if you want to test without calling external APIs, but I don't personally use them much